core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2770,7 +2772,7 @@ public function toggle($intId=null)
 		}
 
 		// Check the field access
-		if (!System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_EDIT_FIELD_OF_TABLE, $this->strTable . '::' . $this->strField))
+		if (!DataContainer::isFieldExcluded($this->strTable, $this->strField) && !System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_EDIT_FIELD_OF_TABLE, $this->strTable . '::' . $this->strField))
```

Comment:
There seems to be no `DataContainer::isFieldExcluded()` method:

<img width="648" alt="" src="https://user-images.githubusercontent.com/1192057/224988831-7c38035c-3822-4bf6-8fcf-5490a220e76b.png">

Reply:
ah you're right, this is Contao 5 only. What's the best approach to remove this and make sure it gets re-added in Contao 5?

Reply:
Can we simply add the method in Contao 4.13, too?

Reply:
I can handle this when I merge the PR, however, are you sure that `!DataContainer::isFieldExcluded()` is correct? It should be `DataContainer::isFieldExcluded()` (without the `!`) IMHO.

Reply:
you're right!